config/routes.rb

```diff
@@ -6,7 +6,8 @@
   get "/learn-more", to: "static_pages#learn_more", as: "learn_more"
   get "/tools-resources", to: "static_pages#tools_resources", as: "tools_resources"
   get "/techhire-locations", to: "static_pages#techhire_locations", as: "techhire_locations"
-  get '/download', to: "static_pages#download", as: "download"
+  get "/download", to: "static_pages#download", as: "download"
+  get "/companies/welcome", to: "companies#welcome"
```

Comment:
I like using this better than sending them to their company show page before their account is confirmed.